tests/sentry/snuba/metrics/test_metrics_layer/test_metrics_enhanced_performance.py

```diff
@@ -240,6 +240,107 @@ def test_alias_on_same_metrics_expression_but_different_aliases(self):
             key=lambda elem: elem["name"],
         )
 
+    def test_custom_measurement_query_with_valid_mri(self):
+        CUSTOM_SPEED_MRI = "d:transactions/measurements.speed@millisecond"
+
+        for value in (100, 200, 300):
+            self.store_metric(
+                org_id=self.organization.id,
+                project_id=self.project.id,
+                type="distribution",
+                name=CUSTOM_SPEED_MRI,
+                tags={},
+                timestamp=int(time.time()),
+                value=value,
+                use_case_id=UseCaseKey.PERFORMANCE,
+            )
+
+        metrics_query = MetricsQuery(
+            org_id=self.organization.id,
+            project_ids=[self.project.id],
+            select=[
+                MetricField(
+                    op="count",
+                    metric_mri=CUSTOM_SPEED_MRI,
+                ),
+            ],
+            start=self.now - timedelta(hours=1),
+            end=self.now,
+            granularity=Granularity(granularity=3600),
+            groupby=[],
+            orderby=[],
+            limit=Limit(limit=2),
+            offset=Offset(offset=0),
+            include_series=False,
+        )
+
+        data = get_series(
+            [self.project],
+            metrics_query=metrics_query,
+            include_meta=True,
+            use_case_id=UseCaseKey.PERFORMANCE,
+        )
+        groups = data["groups"]
+        assert len(groups) == 1
+
+        expected_sum = 3
+        expected_alias = "count(measurements.speed)"
+        assert groups[0]["totals"] == {
+            expected_alias: expected_sum,
+        }
+        assert data["meta"] == sorted(
+            [
+                {"name": expected_alias, "type": "UInt64"},
+            ],
+            key=lambda elem: elem["name"],
+        )
+
+    def test_custom_measurement_query_with_invalid_mri(self):
```

Comment:
This test is redundant so we can remove it as this would fail here https://github.com/getsentry/sentry/blob/a78d8729f96d3d274f854edd59d625972b2656fe/src/sentry/snuba/metrics/query.py#L103 anyways